ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -287,3 +287,7 @@ sh etc/init.d/rc.logging
 
 mavlink boot_complete
 replay trystart
+
+if [ "$PX4_SIM_LABEL" = "rtps" ]; then
```

Comment:
There is no need to hijack the rcS file. You can modify https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d-posix/1016_iris_rtps.post. Also, this has to be documented in the devguide, since you are defining defaults to the ports starting in 3000 and 4000.

Reply:
Thanks. Can I define port 3000 and 4000 for myself? Could you recommend port number?

Reply:
Also, this does mean that the defaults need to be updated in https://github.com/PX4/Firmware/blob/master/src/modules/micrortps_bridge/micrortps_client/microRTPS_client.h#L59-L60 and https://github.com/PX4/Firmware/blob/master/msg/templates/urtps/microRTPS_agent.cpp.em#L84-L85.

Reply:
> Can I define port 3000 and 4000 for myself?

Well you can but then this is not a contribution to the community, right? :) well I think we can use those ports but you need to make the changes on the files above (the defaults) so we know that the common configuration works (meaning with a single vehicle. using the default UDP ports).

Reply:
Yes, I agree. How about (r:2019, s:2020), (r:2021, s:2022), (r:2023, s:2024) ...  ?

Reply:
You can split with the values you are defining already. But it just needs to be updated on the defaults